aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
 
-Configure Data Protection
```

Comment:
... and a good spot to mention in passing that I thought the process follows this sequence (and getting the heading levels striaghtened out) (and subject to change based on your final decisions on section further discussed below) ... 

| Content | Level | Markup |
| ------- | ----- | ------ |
| Title: Publishing to IIS | Title | H1 as = |
| .... Supported Operating Systems | Section | H2 as - |
| ........  Windows Desktop Operating Systems | Subsection | H3 as ^ |
| ........  Windows Server Operating Systems | Subsection | H3 as ^ |
| .... IIS Configuration | Section | H2 as - |
| .... Install .NET Core on the Server | Section | H2 as - |
| .... Deploy the Application | Section | H2 as - |
| .... Configure the Website in IIS | Section | H2 as - |
| .... Create a Data Protection Registry Hive | Section | H2 as - |
| .... Common Errors | Section | H2 as - |
| .... Additional Resources | Section | H2 as - |